app/views/content/is-teaching-right-for-me/maths.md

```diff
@@ -5,7 +5,7 @@ description: |-
 subcategory: What to teach
 navigation: 5.46
 navigation_title: Maths
-navigation_description: Find out how to become a maths teacher and what it's like to help pupils learn to solve equations.
+navigation_description: Find out how to become a maths teacher and show children the importance of maths in everything we do, from cooking and shopping to managing money and making decisions.
```

Comment:
```suggestion
navigation_description: Find out how to become a maths teacher and show children the importance of maths in everything we do.
```
Think this looks a bit wordy now - do we need to go into a lot of detail? Not sure our UR shows that people take a lot of notice of the descriptions!